test-result-summary-client/src/Build/Summary/Overview.jsx

```diff
@@ -27,6 +27,14 @@ export default class Overview extends Component {
 
             const buildName = parentBuildInfo.buildName;
 
+            let warningMsg = '';
+            if (sdkBuilds && sdkBuilds.length === 0) {
+                warningMsg = 'No JDK Builds get triggered in this pipeline. ';
```

Comment:
Minor grammar suggestion to switch the wording to past tense either to got triggered, or were triggered instead of saying get triggered, since the pipeline has already been run.  (for both this and line 36)